src/components/BuildBaronAndAnnotations/AnnotationTickets.tsx

```diff
@@ -1,22 +1,74 @@
-import React from "react";
+import React, { useState } from "react";
+import styled from "@emotion/styled";
+import Tooltip from "antd/es/tooltip";
+import { ConditionalWrapper } from "components/ConditionalWrapper";
+import { PlusButton } from "components/Spawn";
 import { IssueLink } from "gql/generated/types";
+import { AddIssueModal } from "./AddIssueModal";
 import { AnnotationTicketsTable } from "./AnnotationTicketsTable";
 import { TicketsTitle, TitleAndButtons } from "./BBComponents";
 
 interface Props {
+  taskId: string;
+  execution: number;
   tickets: IssueLink[];
-  title: string;
+  isIssue: boolean;
+  userModifyPermission: boolean;
 }
 
-export const AnnotationTickets: React.FC<Props> = ({ tickets, title }) => (
-  <>
-    {tickets?.length && (
-      <>
-        <TitleAndButtons>
-          <TicketsTitle>{title} </TicketsTitle>
-        </TitleAndButtons>
-        <AnnotationTicketsTable jiraIssues={tickets} />{" "}
-      </>
-    )}
-  </>
-);
+export const AnnotationTickets: React.FC<Props> = ({
+  tickets,
+  taskId,
+  execution,
+  isIssue,
+  userModifyPermission,
+}) => {
+  const title = isIssue ? "Issues" : "SuspectedIssues";
+  const buttonText = isIssue ? "Add Issue" : "Add SuspectedIssue";
+
+  const [
+    isAddAnnotationModalVisible,
+    setIsAddAnnotationModalVisible,
+  ] = useState<boolean>(false);
+  return (
+    <>
+      {tickets?.length && (
+        <>
```

Comment:
Lets bring this conditional up so we avoid rendering an empty fragment if `tickets?.length` is undefined.